docs/l3/index.md

```diff
@@ -6,14 +6,43 @@ L3 will demonstrate the value of creating a shared public infrastructure for man
 
-## Process flows
-
-Process flows for transactions in the L3 token model can be found in [`dscp-matchmaker-api`](https://github.com/digicatapult/dscp-matchmaker-api#process-flows). This has also been translated into [our DSL](../tokenModels/language.md) and this can be found [here](./l3.dscp)
+This design has then also been translated to process flow guard rails whose implementation can be found in [`dscp-matchmaker-api`](https://github.com/digicatapult/dscp-matchmaker-api#process-flows).
 
 ## API flows
 
-The API flow for 'dscp-matchmaker-api' can be found in [Match Creation API Flow](api_flow.md)
+The API flows for 'dscp-matchmaker-api' can be found in [API flows](api_flow.md).
+
+## Glossary and transaction summary
+
+This section describes some terminology used in the L3 project.
+
+### Main entities and routes
+
+| name        | description                                                                                                                                         | route prefixes               |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
+| demand      | A demand represents an abstract need for some sort of matchmaking. In L3 this represents either an order or a capacity                              | `/v1/demandA`, `/v1/demandB` |
+| demand_a    | Matches in L3 are between heterogeneous demands and a demand_a represents one side of this                                                          | `/v1/demandA`                |
+| order       | A demand to have a physical asset moved from one location to another. This is represented by a demand_a in L3                                       | `/v1/demandA`                |
+| demand_b    | Matches in L3 are between heterogeneous demands and a demand_b represents the other side of this                                                    | `/v1/demandB`                |
+| capacity    | An ability to move physical assets via some sort of transportation mechanism. This represents a demand to have space filled by orders               | `/v1/demandB`                |
+| match       | An agreement to have an order moved by a specified capacity                                                                                         | `/v1/match2`                 |
+| re-match    | An agreement to have an order moved by a specified capacity which replaces replaces an existing match for an order. This is a special kind of match | `/v1/match2`                 |
+| transaction | An onchain transaction that communicates a change of state for an order, a capacity and/or a match                                                  |
+| attachment  | A file that needs to be communicated over shared infrastructure consistently, but will not affect the validity of a transaction                     | `/v1/attachment`             |
```

Comment:
I may not remember correctly, but if you are cancelling a match2 - you need to provide an attachment. So that does affect the validity of the transaction, because without the attachment you cannot cancel, right?

Reply:
I've reworded to make clear it's the contents of the file that won't affect validity